R/pkg/inst/tests/test_rdd.R

```diff
@@ -141,7 +141,8 @@ test_that("PipelinedRDD support actions: cache(), persist(), unpersist(), checkp
   unpersist(rdd2)
   expect_false(rdd2@env$isCached)
 
-  setCheckpointDir(sc, "checkpoints")
+  tempDir <- tempfile(pattern = "checkpoint")
```

Comment:
It creates a path name, and Spark will create the directory according a specific file system. On my local test, checkpoint seems to work as I can see the checkpoint directory with data. But still, I can create a directory in R, and pass it to the setCheckpointDir().